src/components/messenger/user-profile/index.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+
+import { bemClassName } from '../../../lib/bem';
+
+import { PanelHeader } from '../list/panel-header';
+import { Image } from '@zero-tech/zui/components';
+import { IconCurrencyEthereum } from '@zero-tech/zui/icons';
+
+import './styles.scss';
+
+const cn = bemClassName('user-profile');
+
+export interface Properties {
+  name: string;
+  image: string;
+  subHandle?: string;
+
+  onBack: () => void;
+}
+
+export class UserProfile extends React.Component<Properties> {
+  navigateBack = () => {
```

Comment:
Note : I'll use this in a follow up PR. Currently passing the prop into onBack instead.